src/ipc-handlers.ts

```diff
@@ -1282,7 +1282,7 @@ export async function saveUserTerminal(
 
 export async function getUserTerminal( _event: IpcMainInvokeEvent ): Promise< SupportedTerminal > {
 	const userData = await loadUserData();
-	return userData.supportedTerminal as SupportedTerminal;
+	return ( userData.supportedTerminal as SupportedTerminal ) || DEFAULT_TERMINAL;
```

Comment:
```suggestion
	return userData.supportedTerminal || DEFAULT_TERMINAL;
```
No need for the `as` statement anymore.

Reply:
Adjusted in https://github.com/Automattic/studio/pull/1265/commits/769193f3f79728852a8396f540635c62f5fe2c7a